it/uk/gov/hmrc/emailverification/EmailVerificationISpec.scala

```diff
@@ -15,56 +14,64 @@ class EmailVerificationISpec extends IntegrationBaseSpec with GivenWhenThen {
 
       When("a client submits a verification request")
 
-      val response = appClient("/verification-requests").post(verificationRequest(emailToVerify, templateId, continueUrl)).futureValue
-      response.status shouldBe 201
+      withClient { ws =>
+        val response = await(ws.url(appClient("/verification-requests")).post(verificationRequest(emailToVerify, templateId, continueUrl)))
+        response.status shouldBe 201
 
-      Then("an email is sent")
-      verifyEmailSent(emailToVerify, continueUrl, templateId, paramsWithVerificationLink)
+        Then("an email is sent")
+        verifyEmailSent(emailToVerify, continueUrl, templateId, paramsWithVerificationLink)
+      }
     }
 
     "only latest email verification request token for a given email should be valid" in new Setup {
       Given("The email service is running")
       stubSendEmailRequest(202)
 
       When("client submits a verification request")
-      val response1 = appClient("/verification-requests").post(verificationRequest(emailToVerify, templateId, continueUrl)).futureValue
-      response1.status shouldBe 201
-      val token1 = decryptedToken(lastVerificationEMail)._1.get
-
-      When("client submits a second verification request for same email")
-      val response2 = appClient("/verification-requests").post(verificationRequest(emailToVerify, templateId, continueUrl)).futureValue
-      response2.status shouldBe 201
-      val token2 = decryptedToken(lastVerificationEMail)._1.get
-
-      Then("only the last verification request token should be valid")
-      appClient("/verified-email-addresses").post(Json.obj("token" -> token1)).futureValue.status shouldBe 400
-      appClient("/verified-email-addresses").post(Json.obj("token" -> token2)).futureValue.status shouldBe 201
+      withClient { ws =>
+        val response1 = await(ws.url(appClient("/verification-requests")).post(verificationRequest(emailToVerify, templateId, continueUrl)))
```

Comment:
There are a few of the vals named 'response' that could be could be put in a private val at the top level to reduce duplication.